docs/pages/branding/about.tsx

```diff
@@ -394,31 +394,31 @@ const company = [
   {
     name: 'Sebastian Silbermann',
     src: '/static/branding/about/sebastian.jpg',
-    title: 'Community focus',
+    title: 'Design system focus',
```

Comment:
What's your definition of a design system?

Reply:
I would say https://getbootstrap.com/

Reply:
I can revert the change if you feel it's not correct

Reply:
Reverted